velox/exec/tests/HashJoinTest.cpp

```diff
@@ -1110,6 +1110,42 @@ TEST_F(HashJoinTest, antiJoinWithFilter) {
       "SELECT t.* FROM t WHERE NOT EXISTS (SELECT * FROM u WHERE t0 = u0 AND t1 <> u1)");
 }
 
+TEST_F(HashJoinTest, notInNullWithFilter) {
```

Comment:
There is an existing antiJoinWithFilter test. Would you consider adding to that test or renaming the new test to use 'antiJoin' instead of 'notIn' for consistency?

Reply:
That one is for `NOT EXISTS`.  I will rename that one.